tests/console/parsec.pm

```diff
@@ -0,0 +1,60 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test parsec service with parsec-tool
+# Maintainer: Guillaume Gardet <guillaume@opensuse.org>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+    # Install requirements
+    select_console 'root-console';
+    my $pkg_list = "parsec parsec-tool";
+    zypper_call("in $pkg_list");
+    systemctl 'start parsec';
+
+    # Add user to 'parsec-clients' group and force relogin
+    assert_script_run("usermod -a -G parsec-clients $testapi::username");
+    select_console('user-console');
+    type_string("exit\n");
```

Comment:
```suggestion
    type_string('exit\n');
```

Reply:
No, it will write `exit\n` instead of `exit` + `enter` key, right?

Reply:
Indeed